src/main/scala/org/constellation/ConstellationNode.scala

```diff
@@ -288,6 +290,12 @@ class ConstellationNode(
     api
   }
 
+  def getPeerAPIClient: APIClient = {
+    val api = APIClient(dao.nodeConfig.hostName, dao.nodeConfig.peerHttpPort)
```

Comment:
why do you want to talk to your own peer api?

Reply:
nvm, I see we use it in testing. In some sense both of those should probably be defined in TestNode or something but it's cool for now